meson.build

```diff
@@ -0,0 +1,49 @@
+project('RSDKv3', ['c', 'cpp'], default_options: [
+    'c_std=c17',
+    'cpp_std=c++17',
+    'optimization=2',
+    'prefer_static=true',
+])
+
+executable(
+    'RSDKv3',
+    files(
+        'dependencies/all/tinyxml2/tinyxml2.cpp',
+        'dependencies/all/theoraplay/theoraplay.c',
```

Comment:
Can't these be added as dependencies too? Particularly this should be practical for tinyxml2, which already can be installed as a system dependency -- or via https://mesonbuild.com/Wrap-dependency-system-manual.html and `meson wrap install tinyxml2`.

Reply:
Probably, I was trying to avoid deviating too much from the Makefile, which uses the files as sources.

Should we prefer system dependencies when possible? That would also make `prefer_static=true` questionable.